### PR TITLE
add test for sighting, modify gitignore

### PR DESCRIPTION
**Pull request by mistergreen22**

### CODE REVIEW

File: tests/functional/tests/test_sighting.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        assert sighting[
+                   'external_references'][0][
+                   'url'] == 'https://www.abuseipdb.com/check/1.1.1.1'
+        assert sighting['observed_time']['start_time'] is not None
+        assert sighting['observables'][0] == {'value': '1.1.1.1', 'type': 'ip'}
+        assert sighting[
+                   'relations'][0]['origin'] == 'AbuseIPDB Enrichment Module'
+        assert sighting[
+                   'relations'][0][
+                   'origin_uri'] == 'https://www.abuseipdb.com/check/1.1.1.1'
```
Comment (denyskulykov) [inline tests/functional/tests/test_sighting.py]: Please, make more readable

File: tests/functional/tests/test_sighting.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    """
+    payload = {'type': 'ip', 'value': '1.1.1.1'}
+    response = enrich_observe_observables(
+        payload=[payload],
+        **{'headers': module_headers}
+    )['data']
+    sightings = get_observables(response, 'Abuse IPDB')['data']['sightings']
+    for sighting in sightings['docs']:
+        assert sighting['type'] == 'sighting'
+        assert sighting['schema_version'] == '1.0.16'
```
Comment (denyskulykov) [inline tests/functional/tests/test_sighting.py]: schema_version can change 
Please  just check that key schema_version exists in dictionary

File: tests/functional/tests/test_sighting.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                   'source_uri'] == 'https://www.abuseipdb.com/check/1.1.1.1'
+        assert sighting['confidence'] == 'Medium'
+        assert sighting['title'] == 'Reported to AbuseIPDB'
+        assert sighting['description'] is not None
+        assert sighting['count'] == 5
+        assert sighting['internal'] is False
+        assert sighting['external_references'][0]['source_name'] == 'AbuseIPDB'
+        assert sighting[
+                   'external_references'][0][
+                   'url'] == 'https://www.abuseipdb.com/check/1.1.1.1'
```
Comment (denyskulykov) [inline tests/functional/tests/test_sighting.py]: the same about readable

File: tests/functional/tests/test_sighting.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        assert sighting['observed_time']['start_time'] is not None
+        assert sighting['observables'][0] == {'value': '1.1.1.1', 'type': 'ip'}
+        assert sighting[
```
Comment (denyskulykov) [inline tests/functional/tests/test_sighting.py]: Please, create dict relations and just compare 
assert sighting['relations'][0] ==  relations

File: tests/functional/tests/test_sighting.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
+from ctrlibrary.core.utils import get_observables
+from ctrlibrary.threatresponse.enrich import enrich_observe_observables
+
+
+def test_positive_sighting_ip_observable(module_headers):
+    """Perform testing for enrich observe observables endpoint to get
+    sighting for observable from Abuse IPDB module
+
+    ID: CCTRI-f58e47e4-f00e-423c-9a53-e3d03e337018
```
Comment (denyskulykov) [inline tests/functional/tests/test_sighting.py]: Please add number of story

Comment (mistergreen22) [reply tests/functional/tests/test_sighting.py]: done

File: tests/functional/tests/test_sighting.py (hunk 2 above)
Comment (mistergreen22) [reply tests/functional/tests/test_sighting.py]: done

File: tests/functional/tests/test_sighting.py (hunk 3 above)
Comment (mistergreen22) [reply tests/functional/tests/test_sighting.py]: done

File: tests/functional/tests/test_sighting.py (hunk 1 above)
Comment (mistergreen22) [reply tests/functional/tests/test_sighting.py]: done

File: tests/functional/tests/test_sighting.py (hunk 4 above)
Comment (mistergreen22) [reply tests/functional/tests/test_sighting.py]: done

File: tests/functional/tests/test_sighting.py [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+        payload=[payload],
+        **{'headers': module_headers}
+    )['data']
+    sightings = get_observables(response, 'Abuse IPDB')['data']['sightings']
+    relations = {
+        'origin': 'AbuseIPDB Enrichment Module',
+        'origin_uri': 'https://www.abuseipdb.com/check/1.1.1.1',
+        'relation': 'Resolved_To',
+        'source': {'value': 'cloudflare.com', 'type': 'domain'},
+        'related': {'value': '1.1.1.1', 'type': 'ip'}
```
Comment (oshtaier) [inline tests/functional/tests/test_sighting.py]: `'related': payload`

File: tests/functional/tests/test_sighting.py [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+    }
+    for sighting in sightings['docs']:
+        assert sighting['type'] == 'sighting'
+        assert sighting['schema_version'] is not None
+        assert sighting['source'] == 'AbuseIPDB'
+        assert sighting[
+                   'source_uri'] == 'https://www.abuseipdb.com/check/1.1.1.1'
```
Comment (oshtaier) [inline tests/functional/tests/test_sighting.py]: remove 8 spaces

File: tests/functional/tests/test_sighting.py [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+        assert sighting['confidence'] == 'Medium'
+        assert sighting['title'] == 'Reported to AbuseIPDB'
+        assert sighting['description'] is not None
+        assert sighting['count'] == 5
+        assert sighting['internal'] is False
+        assert sighting['external_references'][0]['source_name'] == 'AbuseIPDB'
+        assert sighting['external_references'][0]['url'] == (
+            'https://www.abuseipdb.com/check/1.1.1.1')
+        assert sighting['observed_time']['start_time'] is not None
+        assert sighting['observables'][0] == {'value': '1.1.1.1', 'type': 'ip'}
```
Comment (oshtaier) [inline tests/functional/tests/test_sighting.py]: `assert sighting['observables'][0] == payload`

File: tests/functional/tests/test_sighting.py [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+    relations = {
+        'origin': 'AbuseIPDB Enrichment Module',
+        'origin_uri': 'https://www.abuseipdb.com/check/1.1.1.1',
+        'relation': 'Resolved_To',
+        'source': {'value': 'cloudflare.com', 'type': 'domain'},
+        'related': {'value': '1.1.1.1', 'type': 'ip'}
+    }
+    for sighting in sightings['docs']:
+        assert sighting['type'] == 'sighting'
+        assert sighting['schema_version'] is not None
```
Comment (oshtaier) [inline tests/functional/tests/test_sighting.py]: `assert sighting['schema_version']`

File: tests/functional/tests/test_sighting.py [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+        assert sighting['source'] == 'AbuseIPDB'
+        assert sighting[
+                   'source_uri'] == 'https://www.abuseipdb.com/check/1.1.1.1'
+        assert sighting['confidence'] == 'Medium'
+        assert sighting['title'] == 'Reported to AbuseIPDB'
+        assert sighting['description'] is not None
```
Comment (oshtaier) [inline tests/functional/tests/test_sighting.py]: `assert sighting['description']`

File: tests/functional/tests/test_sighting.py [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+        assert sighting['count'] == 5
+        assert sighting['internal'] is False
+        assert sighting['external_references'][0]['source_name'] == 'AbuseIPDB'
+        assert sighting['external_references'][0]['url'] == (
+            'https://www.abuseipdb.com/check/1.1.1.1')
+        assert sighting['observed_time']['start_time'] is not None
```
Comment (oshtaier) [inline tests/functional/tests/test_sighting.py]: `assert sighting['observed_time']['start_time']`

**Comment by oshtaier** [comment]
and of course fix travis

File: tests/functional/tests/test_sighting.py (hunk 6 above)
Comment (mistergreen22) [reply tests/functional/tests/test_sighting.py]: done

File: tests/functional/tests/test_sighting.py (hunk 7 above)
Comment (mistergreen22) [reply tests/functional/tests/test_sighting.py]: change formatting with ()

File: tests/functional/tests/test_sighting.py (hunk 8 above)
Comment (mistergreen22) [reply tests/functional/tests/test_sighting.py]: done

File: tests/functional/tests/test_sighting.py (hunk 9 above)
Comment (mistergreen22) [reply tests/functional/tests/test_sighting.py]: done

File: tests/functional/tests/test_sighting.py (hunk 10 above)
Comment (mistergreen22) [reply tests/functional/tests/test_sighting.py]: done

File: tests/functional/tests/test_sighting.py (hunk 11 above)
Comment (mistergreen22) [reply tests/functional/tests/test_sighting.py]: done

**Pull request merged**